src/test/java/org/folio/qm/validation/leader/AuthorityLeaderValidationRuleTest.java

```diff
@@ -0,0 +1,66 @@
+package org.folio.qm.validation.leader;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+
+import org.hamcrest.core.Is;
+import org.junit.jupiter.api.Test;
+
+import org.folio.qm.validation.LeaderValidationRule;
+import org.folio.qm.validation.ValidationError;
+import org.folio.qm.validation.impl.authority.AuthorityLeaderValidationRule;
+
+class AuthorityLeaderValidationRuleTest {
+
+  private LeaderValidationRule rule = new AuthorityLeaderValidationRule();
+  public static final String VALID_LEADER = "00241xz\\\\a2200109nu\\4500";
+  public static final String WRONG_RECORD_STATUS = "00241zz\\\\a2200109nu\\4500";
+  public static final String WRONG_RECORD_TYPE = "00241xx\\\\a2200109nu\\4500";
+  public static final String WRONG_UNDEFINED_CHARACTER = "00241xzz\\a2200109nu\\4500";
+  public static final String WRONG_ENCODING_LEVEL = "00241xz\\\\a2200109au\\4500";
+  public static final String WRONG_PUNCTUATION_POLICY = "00241xz\\\\a2200109nz\\4500";
```

Comment:
This could be private

Reply:
in this case, please, marc as private fields for these class too: `BibliographicLeaderValidationRuleTest`, `CommonLeaderValidationRuleTest`, `HoldingsLeaderValidationRuleTest`